pilot/pkg/features/telemetry.go

```diff
@@ -51,4 +51,8 @@ var (
 
 	StackdriverAuditLog = env.Register("STACKDRIVER_AUDIT_LOG", false, ""+
 		"If enabled, StackDriver audit logging will be enabled.").Get()
+
+	// DisableBootstrapTracing disables the bootstrap tracing configuration. Will make the configuration more like Telemetry API.
+	DisableBootstrapTracing = env.Register("DISABLE_BOOTSTRAP_TRACING", false,
```

Comment:
Anthos service mesh uses the bootstrap tracing configuration (https://cloud.google.com/service-mesh/docs/managed/enable-managed-anthos-service-mesh-optional-features). Setting DISABLE_BOOTSTRAP_TRACING as false by default looks good.

Reply:
This PR doesn't really change that? It just is an implementation detail on _how_ those configs are applied. Assuming this PR works correctly, it should work fine.

And it will be `true` eventually (or we should not merge it), so I wouldn't approve just because its `false`.

Reply:
it will  break users who use `zipkin` with app outside the cluster and there isn't a ServiceEntry, that's why I make it default `false`.

Reply:
Maybe we can set it to false in compatbility-version=1.21 (or in 1.22)